BigQueryDataTransfer/src/V1/DataTransferServiceClient.php

```diff
@@ -31,6 +31,73 @@
  */
 class DataTransferServiceClient extends DataTransferServiceGapicClient
 {
-    // This class is intentionally empty, and is intended to hold manual
-    // additions to the generated {@see DataTransferServiceGapicClient} class.
+    /**
+     * Formats a string containing the fully-qualified path to represent
+     * a location_data_source resource.
+     *
+     * @param string $project
+     * @param string $location
+     * @param string $dataSource
+     *
+     * @return string The formatted location_data_source resource.
+     *
+     * @deprecated Multi-pattern resource names will have unified formatting functions.
+     *             This helper function will be deleted in the next major version.
+     */
+    public static function locationDataSourceName($project, $location, $dataSource)
+    {
+        return self::projectLocationDataSourceName($project, $location, $dataSource);
+    }
+    /**
```

Comment:
This is the nittiest of nitpicks, so feel free to ignore - but this file doesn't have spaces between the method declarations :).